examples/file_browser.html

```diff
@@ -0,0 +1,119 @@
+<!--
+
+   Copyright (c) 2020, the Regular Table Authors.
+
+   This file is part of the Regular Table library, distributed under the terms of
+   the Apache License 2.0.  The full license can be found in the LICENSE file.
+
+-->
+
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
+    <script src="../umd/regular-table.js"></script>
+    <link rel='stylesheet' href="../css/material.css">
+</head>
+
+<body>
+
+    <regular-table></regular-table>
+
+    <script>
+
+        const NUM_ROWS = 100;
+        const NUM_DIR = 10;
+        const NUM_FILE = NUM_ROWS - NUM_DIR;
+
+        const COLUMN_NAMES = ['modified', 'writable'];
+
+        const formatter = new Intl.DateTimeFormat("en-us");
+
+        const _contents_cache = new Map();
+        function getContents(path, expand) {
```

Comment:
This is clever - but it would benefit from being cleaned up, and maybe structurally separated to a logically distinct `generateFakeFilesystem()` method so its more clear how this examples uses the `regular-table` API.

Reply:
I'm fine with the name change, but I don't quite follow the rest of your comment here. Can you please add some more details and/or concrete examples of what you'd like to see changed?